build-utils/run_ci.sh

```diff
@@ -22,6 +22,11 @@ if [[ -z $COMMAND || -z $STUDY_KEY || -z $BRANCH || ($COMMAND == "deploy" && -z
   echo "        Deploy saved build of corresponding to given branch to specified TARGET_ENV"
   echo "    run-tests"
   echo "        Run tests for given study and given branch"
+  echo "    run-e2e-tests"
+  echo "        Run Playwright E2E tests for given branch against specified TARGET_ENV (dev, test or staging)"
+  echo "        Given study is ignored"
```

Comment:
Can you please clarify what does "Given study is ignored" mean here?

Reply:
See added explanation in README. API triggers a call to run all tests, not limited to any one or more studies.